src/utils.py

```
@@ -0,0 +1,93 @@
+    "manager",
+    "director",
+    "executive",
+    "analyst",
+    "consultant",
+    "specialist",
+    "technician",
+    "administrator",
+]
+
+
+def generate_random_name():
```

Ruff (0.8.2)

Missing return type annotation for public function `generate_random_name`

Add return type annotation: `str`

(ANN201)